### PR TITLE
Updates readme for windows installer

### PR DESCRIPTION
**Pull request by betarelease**
- elaborates on where to find files
- refactors the flow of actions
- fixes the debug flag

## Description

Fixes pyrsia/pyrsia#

This PR updates readme for windows installer - to fix a few things that were confusing to me when I installed.

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### CODE REVIEW

**Review by erwin1 — DISMISSED** [review]

File: installers/windows/Readme.txt [hunk 1]
```diff
@@ -18,35 +18,37 @@ The MSI installer can be executed by double-clicking on it, but it is more conve
...
-Connected Peers Count:       1
+and also check the log file C:\tmp\log.txt if needed. The log file can help you troubleshoot if the installation did not succeed.
 
 The service folder contains the pyrsia_node executable that can be launched from any terminal.
 
 For convenience, set these environment variables:
 
 > set DEV_MODE=ON
-> set RUST_LOG=pyrsia=debug
+> set RUST_LOG=debug
```
Comment (fishseabowl) [inline installers/windows/Readme.txt:35]: If we only output Pyrsia debug logs, it's better to use `> set RUST_LOG=pyrsia=debug`

**Review by fishseabowl — DISMISSED** [review]

File: installers/windows/Readme.txt (hunk 1 above)
Comment (betarelease) [reply installers/windows/Readme.txt]: updated based on your comment

**Review by erwin1 — DISMISSED** [review]

**Review by erwin1 — APPROVED** [review]

**Review by dasmanas — DISMISSED** [review]
lgtm

File: installers/windows/Readme.txt [hunk 2]
```diff
@@ -18,35 +18,37 @@ The MSI installer can be executed by double-clicking on it, but it is more conve
...
+C:\Pyrsia\Pyrsia\service> pyrsia_node.exe
  2022-10-03T17:54:20.328Z DEBUG pyrsia_node > Parse CLI arguments
  2022-10-03T17:54:20.329Z DEBUG pyrsia_node > Create p2p components
  2022-10-03T17:54:20.336Z DEBUG pyrsia_node > Start p2p event loop
  2022-10-03T17:54:20.337Z DEBUG pyrsia_node > Create blockchain service component
 ...
 
+In another command prompt
+
+The pyrsia CLI(C:\Pyrsia\Pyrsia\bin\pyrsia.exe) is added to the system PATH.
```
Comment (jperedadnr) [inline installers/windows/Readme.txt:47]: minor: add blank space after `CLI`

File: installers/windows/Readme.txt [hunk 3]
```diff
@@ -18,35 +18,37 @@ The MSI installer can be executed by double-clicking on it, but it is more conve
...
-
-> pyrsia_node.exe
+C:\Pyrsia\Pyrsia\service> pyrsia_node.exe
  2022-10-03T17:54:20.328Z DEBUG pyrsia_node > Parse CLI arguments
  2022-10-03T17:54:20.329Z DEBUG pyrsia_node > Create p2p components
  2022-10-03T17:54:20.336Z DEBUG pyrsia_node > Start p2p event loop
  2022-10-03T17:54:20.337Z DEBUG pyrsia_node > Create blockchain service component
 ...
 
+In another command prompt
```
Comment (jperedadnr) [inline installers/windows/Readme.txt:45]: Remove this line?

File: installers/windows/Readme.txt [hunk 4]
```diff
@@ -18,35 +18,37 @@ The MSI installer can be executed by double-clicking on it, but it is more conve
...
+
+The pyrsia CLI(C:\Pyrsia\Pyrsia\bin\pyrsia.exe) is added to the system PATH.
+If the service pyrsia_node started correctly and was able to connect to the Pyrsia network you will see the following:
```
Comment (jperedadnr) [inline installers/windows/Readme.txt:48]: If the service pyrsia_node started correctly and was able to connect to the Pyrsia network to the Pyrsia network, run, from another command prompt, the following to verify the same:

File: installers/windows/Readme.txt (hunk 3 above)
Comment (betarelease) [reply installers/windows/Readme.txt:45]: Yes

File: installers/windows/Readme.txt (hunk 4 above)
Comment (betarelease) [reply installers/windows/Readme.txt:48]: rephrased.

**Review by erwin1 — APPROVED** [review]

File: installers/windows/Readme.txt [hunk 5]
```diff
@@ -18,35 +18,36 @@ The MSI installer can be executed by double-clicking on it, but it is more conve
...
  2022-10-03T17:54:20.336Z DEBUG pyrsia_node > Start p2p event loop
  2022-10-03T17:54:20.337Z DEBUG pyrsia_node > Create blockchain service component
 ...
 
+If the service pyrsia_node started correctly and was able to connect to the Pyrsia network you should be able to see connected peers.
+
+From another command prompt, the following to verify the same:
+> pyrsia -s
+Connected Peers Count:       1
+(The pyrsia CLI (C:\Pyrsia\Pyrsia\bin\pyrsia.exe) is added to the system PATH, so this should work. If not find the path to pyrsia.exe and run it like above.)
```
Comment (fishseabowl) [inline installers/windows/Readme.txt:50]:
```suggestion
(The pyrsia CLI (C:\Pyrsia\Pyrsia\bin\pyrsia.exe) is added to the system PATH, so this should work. If not, find the path to pyrsia.exe and run it like above.)
```

**Review by fishseabowl — APPROVED** [review]

**Review by dasmanas — APPROVED** [review]
lgtm

**Pull request merged**